build/gulp/tasks/test-projects.tsx

```diff
@@ -16,11 +17,33 @@ const log = (context: string) => (message: string) => {
   console.log('='.repeat(80))
 }
 
-export const publishPackage = async () => {
-  const filename = tmp.tmpNameSync({ prefix: 'stardust-', postfix: '.tgz' })
-  await sh(`yarn pack --filename ${filename}`)
+const installStardustPackages = async (tmpDirectory: string) => {
+  const stardustPackages = lernaAliases({ sourceDirectory: false })
```

Comment:
nit: comment with the intent of specifying this option would be helpful, even simple like

```
// packages/react/src -> packages/react, as 'src' is appended by default
```